src/libs/actions/Session/index.js

```diff
@@ -343,6 +343,11 @@ function invalidateCredentials() {
     Onyx.merge(ONYXKEYS.CREDENTIALS, {autoGeneratedLogin: '', autoGeneratedPassword: ''});
 }
 
+function invalidateAuthToken() {
+    NetworkStore.setAuthToken('pizza');
+    Onyx.merge(ONYXKEYS.SESSION, {authToken: 'pizza'});
```

Comment:
`pizza`? lol, did you mean to clear it here instead or actually leave it as pizza?

Reply:
🍕  if we set it as null the user would be logged out or other weird things. So "bad authToken" vs. "no authToken" is what we want.

Reply:
cool, can that be added as a comment so that its not easily forgotten for the next person updating this block.